nengo/utils/builder.py

```diff
@@ -177,8 +177,8 @@ def _create_replacement_connection(c_in, c_out):
     return c
 
 
-def remove_passthrough_nodes(objs, connections,  # noqa: C901
-        create_connection_fn=_create_replacement_connection):
```

Comment:
I'm not sure if it was ever okay; the rule is basically you have to do one of

```python
def my_long_func(a, b,
                 c, d):
```
or
```python
def my_long_func(
        a, b, c, d):
```

I think the `# noqa` was being applied for all rules and wasn't just ignoring C901.